readme.md

```diff
@@ -22,10 +22,14 @@ There may be specific steps that are required prior to running these standard st
 
 
 ## Installation
-'*' is a command line tool that comes as a python package.
-Create a virtual env however you prefer, and then `pip install *`
+'commons-boundaries' is a collection of command line tools that come as a python package.
+Create a virtual env however you prefer, and then `pip install commons-boundaries`
 Commands are run at the command line form within the virtual env.
 
+__development__
```

Comment:
A tiny thing, but I think this is really a heading, so I'd do it with `###` which has the right semantics.